src/renderers/shared/fiber/ReactFiberCompleteWork.js

```diff
@@ -251,6 +256,10 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
       case Fragment:
         transferOutput(workInProgress.child, workInProgress);
         return null;
+      case Portal:
+        transferOutput(workInProgress.child, workInProgress);
```

Comment:
Let's make this always leave `output` as `null` and just use this fiber as is in commit work instead.